src/similarities/tanimoto.ts

```diff
@@ -2,10 +2,10 @@ import { NumberArray } from 'cheminfo-types';
 /**
  *Returns the Tanimoto similarity  between vectors p and q, and accepts the bitVector use, see the test case for an example
  * @link [Tanimoto similarity algorithm](https://www.naun.org/main/NAUN/ijmmas/mmmas-49.pdf)
- * @param {NumberArray} a - first vector
- * @param {NumberArray} b - second vector
+ * @param a - first vector
+ * @param b - second vector
  * @param {boolean} bitvector - bitVector
```

Comment:
You have only removed the type annotations for `NumberArray`. That's why I hinted to using a regular expression (that way you can replace all of them regardless of the type)

Reply:
I thought about removing them everywhere, but since there are a few functions with some irregular parameters, I thought it's worth leaving their types there. Should I remove them anyway?